api/pom.xml

```diff
@@ -59,6 +71,7 @@
                         <include>**/*</include>
                     </includes>
                     <excludes>
+                        <exclude>**/AsyncConfiguration.class</exclude>
```

Comment:
why do you have to exclude this?

Reply:
@angshu Removed this exclusion and added a test case for the same.